---
title: "Evolutionary analyses of the major variant surface antigen-encoding genes reveal population structure of Plasmodium falciparum within and between continents"
author: "Gerry Tonkin-Hill, Shazia Ruybal-Pesántez, Kathryn E. Tiedje, Virginie Rougeron, Michael F. Duffy, Sedigheh Zakeri, Tepanata Pumpaibool, Pongchai Harnyuttanakorn, OraLee H. Branch, Lastenia Ruiz-Mesía, Thomas S. Rask, Franck Prugnolle, Anthony T. Papenfuss, Yao-ban Chan, Karen P. Day"
date: '2021-02-25'
slug: var-global
alias:
  - /publications/2021-PLOSGen/
categories:
  - Research
tags:
doi: 'https://doi.org/10.1371/journal.pgen.1009269'
publishDate: '2021-02-25'
publication_types:
  - '2'
excerpt: 'We described population structure at of the highly diverse genes encoding the immunodominant surface antigen of *P. falciparum* at the global scale. This work was chosen by the PLOS Genetics editors to be featured with an accompanying [Perspectives piece.](https://doi.org/10.1371/journal.pgen.1009344)'
subtitle: 'We described population structure at of the highly diverse genes encoding the immunodominant surface antigen of P. falciparum at the global scale'
featured: yes
links:
- icon: doi
  icon_pack: ai
  name: Publication
  url: https://doi.org/10.1371/journal.pgen.1009269
- icon: quote-left
  icon_pack: fas
  name: Accompanying perspectives piece
  url: https://doi.org/10.1371/journal.pgen.1009344
---

**This work was chosen by the PLOS Genetics editors to be featured with an accompanying [Perspectives piece](https://doi.org/10.1371/journal.pgen.1009344)**

## Abstract 

Malaria remains a major public health problem in many countries. Unlike influenza and HIV, where diversity in immunodominant surface antigens is understood geographically to inform disease surveillance, relatively little is known about the global population structure of PfEMP1, the major variant surface antigen of the malaria parasite Plasmodium falciparum. The complexity of the *var* multigene family that encodes PfEMP1 and that diversifies by recombination, has so far precluded its use in malaria surveillance. Recent studies have demonstrated that cost-effective deep sequencing of the region of *var* genes encoding the PfEMP1 DBLα domain and subsequent classification of within host sequences at 96% identity to define unique DBLα types, can reveal structure and strain dynamics within countries. However, to date there has not been a comprehensive comparison of these DBLα types between countries. By leveraging a bioinformatic approach (jumping hidden Markov model) designed specifically for the analysis of recombination within var genes and applying it to a dataset of DBLα types from 10 countries, we are able to describe population structure of DBLα types at the global scale. The sensitivity of the approach allows for the comparison of the global dataset to ape samples of *Plasmodium Laverania* species. Our analyses show that the evolution of the parasite population emerging out of Africa underlies current patterns of DBLα type diversity. Most importantly, we can distinguish geographic population structure within Africa between Gabon and Ghana in West Africa and Uganda in East Africa. Our evolutionary findings have translational implications in the context of globalization. Firstly, DBLα type diversity can provide a simple diagnostic framework for geographic surveillance of the rapidly evolving transmission dynamics of *P. falciparum*. It can also inform efforts to understand the presence or absence of global, regional and local population immunity to major surface antigen variants. Additionally, we identify a number of highly conserved DBLα types that are present globally that may be of biological significance and warrant further characterization.
